tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -43,13 +43,14 @@ apt-get update && apt-get install -y --no-install-recommends \
         bzip2 \
         unzip \
         zip \
-        rsync libunwind8 libpng16-dev \
-        python3-setuptools python3-numpy python3-wheel python python3-pip python3-pytest
+        rsync libunwind8 libpng16-dev libexpat1-dev \
+        python3-setuptools python3-numpy python3-wheel python python3-pip python3-pytest \
+        libprotobuf-dev libprotobuf9v5 protobuf-compiler
 
 locale-gen en_US.UTF-8
 update-locale LANG=en_US.UTF-8
 
-#Install dotnet-sdk
+echo "Installing dotnet-sdk"
```

Comment:
Is there a x86 version of dotnet-sdk for Linux? It is installed for Windows, but not present on Linux.

Reply:
I didn't find it.

Reply:
Looks like it doesn't exist on the official download page either https://dotnet.microsoft.com/download, for .Net Core 2.2. Closing this out.